docs/custom_schemas/h5md/explanation-H5MD-NOMAD.md

```diff
@@ -0,0 +1,1342 @@
+            [J. Appl. Phys. **52**, 7182 (1981)](https://doi.org/10.1063/1.328693);
+            S. Nosé and M.L. Klein, [Mol. Phys. **50**, 1055 (1983) |
+
+            | `"stochastic_cell_rescaling"` | M. Bernetti and G. Bussi,
+            [J. Chem. Phys. **153**, 114107 (2020)](https://doi.org/10.1063/1.2408420) |
+            ''')
+
+* `coupling_type`
+:
+
+        Quantity(
+            type=MEnum('isotropic', 'semi_isotropic', 'anisotropic'),
+            shape=[],
+            description='''
+            Describes the symmetry of pressure coupling. Specifics can be inferred from the `coupling constant`
+
+            | Type          | Description                               |
+
+            | ---------------------- | ----------------------------------------- |
+
+            | `isotropic`          | Identical coupling in all directions. |
+
+            | `semi_isotropic` | Identical coupling in 2 directions. |
+
+            | `anisotropic`        | General case. |
+            ''')
+
+* `reference_pressure`
+:
+
+        Quantity(
+            type=np.float64,
+            shape=[3, 3],
+            unit='pascal',
+            description='''
+            The target pressure for the simulation, stored in a 3x3 matrix, indicating the values for individual directions
+            along the diagonal, and coupling between directions on the off-diagonal.
+            ''')
+
+* `coupling_constant`
+:
+
+        Quantity(
+            type=np.float64,
+            shape=[3, 3],
+            unit='s',
+            description='''
+            The time constants for pressure coupling, stored in a 3x3 matrix, indicating the values for individual directions
+            along the diagonal, and coupling between directions on the off-diagonal. 0 values along the off-diagonal
+            indicate no-coupling between these directions.
+            ''')
+
+* `compressibility`
+:
+
+        Quantity(
+            type=np.float64,
+            shape=[3, 3],
+            unit='1 / pascal',
+            description='''
+            An estimate of the system's compressibility, used for box rescaling, stored in a 3x3 matrix indicating the values for individual directions
+            along the diagonal, and coupling between directions on the off-diagonal. If None, it may indicate that these values
+            are incorporated into the coupling_constant, or simply that the software used uses a fixed value that is not available in
+            the input/output files.
+            ''')
+
+
+
+## Units
+
+In the original H5MD schema, units were given as string attributes of datasets, e.g., ``60 m s-2``.
+H5MD-NOMAD amends the treatment of units in 2 ways:
+
+1. If needed, the leading prefactor is stored as a separate attribute of `float` datatype called `unit_factor`.
+
+2. The string that describes the unit should be compatible with the `UnitRegistry` class of the `pint` python module.
```

Comment:
Maybe explain that NOMAD has its own definitions in nomad.units following the UnitRegistry from pint? I am not 100% sure if pint covers all the possibilities as we do (i.e., I am not sure if nomad/units/default_en.txt and everything defined there is covered by pint, or there are conversions that nomad does which are used for it).

Reply:
I am also not sure, but I think we are safe in the sense that I only am asking that the strings are compatible with the pint package. Also, there is no documentation really about the list of NOMAD unit strings, but there is on the pint docs. And I would sort of like to stay as agnostic to NOMAD as possible, so people aren't overwhelmed by trying to look into NOMAD docs.